src/helpers/url.test.ts

```diff
@@ -0,0 +1,25 @@
+import { isValidUrl } from './url';
+
+describe('URL helper', () => {
+  describe('isValidUrl', () => {
+    describe('when URL is valid', () => {
+      it('returns true', () => {
+        const url = 'https://example.com';
+
+        const isValid = isValidUrl(url);
+
+        expect(isValid).toBe(true);
+      });
+    });
+
+    describe('when URL is invalid', () => {
```

Comment:
```suggestion
    describe('when URL is NOT valid', () => {
```
This one is also @hoangmirs 
I think we can emphasize this context is not a happy case.

Reply:
Changed in https://github.com/nimblehq/til-web/pull/38/commits/62f21752e9e19cae6fcca925a45aa2315819ab18